styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerTest.java

```diff
@@ -91,22 +89,21 @@
   @Mock(answer = RETURNS_DEEP_STUBS) private Container gkeClient;
   @Mock private Container.Projects.Locations.Clusters.Get gkeClusterGet;
   @Mock private Storage storage;
-  @Mock private StorageTransaction transaction;
   @Mock private PersistentStateManager stateManager;
   @Mock private Supplier<Map<WorkflowId, Workflow>> workflowCache;
   @Mock private RateLimiter submissionRateLimiter;
   @Mock private Stats stats;
   @Mock private Time time;
 
-  private StyxScheduler styxScheduler;
+  public StyxSchedulerTest() {}
 
   @Before
   public void setUp() throws Exception {
     MockitoAnnotations.initMocks(this);
     when(kubernetesClientFactory.apply(any(), any())).thenReturn(kubernetesClient);
     when(gkeClient.projects().locations().clusters().get(any())).thenReturn(gkeClusterGet);
 
-    styxScheduler = StyxScheduler.newBuilder().build();
+    StyxScheduler.newBuilder().build();
```

Comment:
Why do we need this scheduler since it is not used at all? Can we remove it?

Reply:
I guess we want to verify it can actually build. Making it a dedicated test case seems better.